sub_workflows/kfdrc_annot_vcf_sub_wf.cwl

```diff
@@ -14,21 +14,26 @@ inputs:
   add_common_fields: {type: boolean, doc: "Set to true if input is a strelka2 vcf that hasn't had common fields added", default: false}
   bcftools_annot_columns: {type: string, doc: "csv string of columns from annotation to port into the input vcf, i.e INFO/AF"}
   bcftools_annot_vcf: {type: File, secondaryFiles: ['.tbi'], doc: "bgzipped annotation vcf file"}
+  bcftools_public_filter: {type: string?, doc: "Will hard filter final result to create a public version"}
```

Comment:
Is the default filter from the temp wf not desired here? Even if it's not it might be nice to provide it as an example in a doc, the current one I think doesn't make it clear that this argument would be a filtering expression.

Reply:
Ah, hmmm. I was debugging and took it out of there, spooked that something wasn't propagating properly. I;ll toss it in there before I merge. Thanks!